packages/studio-web/src/app/demo/demo.component.ts

```diff
@@ -27,6 +35,7 @@ export class DemoComponent implements OnDestroy, OnInit {
   };
   outputFormats = [
     { value: "html", display: $localize`Offline HTML` },
+    { value: "zip", display: $localize`Web Bundle` },
```

Comment:
Zipped Web Bundle?
That would give me "Fichiers Web comprimés" and "Archivos web comprimidos".
I find just "Web Bundle" insufficiently clear.

Reply:
I did some research on "zippés" vs "comprimés" for the French translation, and Google tells me the former is much more common than the latter, and that matches my intuition, but Spanish doesn't seem to have that, it would have to be "Archivos web comprimidos zip" if we wanted to be precise, but I think that's unnecessary, adding more verbosity with no additional clarity, whereas the French "zippés" is shorter and more common, thus a no brainer.

Reply:
compressés: https://vitrinelinguistique.oqlf.gouv.qc.ca/fiche-gdt/fiche/8399261/compresser

Reply:
mais même selon l'OQLF "zippé" est acceptable dans certains contextes ;-)

Reply:
I'm not going to be picky, but my first choice is "zippé" because we're creating an actual .zip file.

Does anyone have an opinion on the English, though, whether we should add "Zipped" in front?

Reply:
I think "zipped" is redundant - Bundle implies that it's bundled/zipped which is sufficient